lib/Layout/Resolver/TargetType/Taxon.php

```diff
@@ -10,6 +10,9 @@
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\Validator\Constraints;
 
+/**
+ * @deprecated this class will be renamed to TaxonTree in next major release
```

Comment:
Please use concrete version: `in 2.0`

Reply:
@emodric Fixed in https://github.com/netgen-layouts/layouts-sylius/pull/17/commits/057d41669cd3128b87eaf8063b07be5151bc5382